### PR TITLE
Add custom color picker

### PR DESCRIPTION
**Pull request by chelliah**
I screwed up the commit history, This PR adds a color picker with an input for hex/rgb
screen shot 2017-11-17 at 2 23 07 pm
 values

### CODE REVIEW

**Comment by trisweb** [comment]
NBD but design feedback: can we collapse the space above a bit? I actually like the tight margins, but should be the same on top as around the sides, then less below the title.

Otherwise: this is awesome.

**Comment by chelliah** [comment]
Agree 100% on the space above. I think all the menu's in the wysiwyg could use some refinement

**Comment by chelliah** [comment]
screen shot 2017-11-20 at 5 32 06 pm
updated

**Comment by trisweb** [comment]
Looks good! Thanks.

File: src/components/ColorPicker.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+    const { inputVal } = this.state;
+    const { onChange } = this.props;
+
+    return (
+      <section>
+          <section style={{position: 'relative', height: '80px', margin: '4px 0'}}>
+            <Saturation
+              {...this.props}
+              onChange={ (val) => this.debounce(onChange, val) }
```
Comment (rayd) [inline src/components/ColorPicker.js]: Is debouncing these onChange calls going to potentially cause us to miss the "last" update in a quick sequence of updates? Or will it always process the last call?

**Pull request merged**